server/controllers/valuesController.js

```diff
@@ -1,11 +1,55 @@
 const { db } = require('../dbConfig');
 
 const valuesController = {
```

Comment:
Not a big deal, but I'd say consider adding "company" where there's only values now. It's not hard to figure out what this does given the context but isn't clear at a quick glance.

Reply:
👍